### ISSUE TITLE
[Request]: add  spawn (  ADD-ON ) vehicles

### ISSUE THREAD

**Issue by D-Borkan** [Yimura/YimMenu#2343]
### Problem

.

### Solution

.

### Reason

It's nice to be in Yimmenu

### Additional context

This feature is available in Stand menu

**Comment by Yimura** [comment]
You'll have to clarify on what these "ADD-ON" vehicles are, as far as I'm aware we support XML and JSON custom vehicle presets.

Could you link to one of these ADD-ON vehicles you're talking about so we can get some much needed context?

**Comment by D-Borkan** [comment]
> You'll have to clarify on what these "ADD-ON" vehicles are, as far as I'm aware we support XML and JSON custom vehicle presets.
> 
> Could you link to one of these ADD-ON vehicles you're talking about so we can get some much needed context?

like this  : https://www.gta5-mods.com/vehicles

**Comment by Yimura** [comment]
The request here is basically to add support for custom mod loading more than just add-on vehicles in general.

**Comment by D-Borkan** [comment]
yeah i tested alot of them and  also work fine even in online

**Comment by ghost** [comment]
I don't understand why people want this feature, as it's not visible to others. However, if ScriptHookV support were available, you could use OpenIV's ASI mod to add addon vehicles and other ASI content. Still, I'm unsure why some mod menus support ScriptHookV, considering that ASI mods are not meant for multiplayer and can be easily detected. For now, you can use a patched version of ScriptHookV to use OpenIV ASI scripts online. [Here's a link, but be cautious, as it's likely detected.](https://www.unknowncheats.me/forum/grand-theft-auto-v/605263-scripthookv-online-enabled.html)

**Comment by D-Borkan** [comment]
> I don't understand why people want this feature, as it's not visible to others. However, if ScriptHookV support were available, you could use OpenIV's ASI mod to add addon vehicles and other ASI content. Still, I'm unsure why some mod menus support ScriptHookV, considering that ASI mods are not meant for multiplayer and can be easily detected. For now, you can use a patched version of ScriptHookV to use OpenIV ASI scripts online. [Here's a link, but be cautious, as it's likely detected.](https://www.unknowncheats.me/forum/grand-theft-auto-v/605263-scripthookv-online-enabled.html)

even if not seen by other but it s an enjoyable experience without need to use  ScriptHookV or asi mod.

**Comment by xiaoxiao921** [comment]
Yeah this is just for convenience. I'm pretty sure everyone from the team is ok to have this into Yim. It's just a matter of someone doing it

**Comment by eggrolls-repu** [comment]
I mean
ok
hmm
Unrelated but it’d be nice if there was a yim crew and a bunch of us could use addon/replacement cars online. But also idk I guess that’s what fivem is for. Mehhh

**Comment by 7HA7MA77** [comment]
@gir489returns - and whole thread is just duplicate, nothing else, just duplicated idea - you not closed it - when in my 10 bugs report 1/10 was duplicate, and you deleted my thread? grow up gir, dont act like a child, it's github of project, it's for community, to make project better, its not your personal space, its not ur facebook, it's a project github..

**Comment by ghost** [comment]
Lol

**Comment by Tessio** [comment]
I think he means to implement the Custom DLCs feature that some paid mod menus have.

This function allows you to load add-on vehicles, weapons, peds, etc.

Examples:
![image](https://github.com/YimMenu/YimMenu/assets/50866332/9360b3aa-17c9-46af-937b-773ce3879031)

![image](https://github.com/YimMenu/YimMenu/assets/50866332/315599cd-a7df-4325-9831-1b0f676f444c)

And another very interesting function is that it modifies the Hash to be able to join an online session with players while using addons. 

![image](https://github.com/YimMenu/YimMenu/assets/50866332/0b686fe2-499b-411c-a9e9-fe8ca789cab4)

Hopefully someone can implement it ♥

**Comment by D-Borkan** [comment]
> I think he means to implement the Custom DLCs feature that some paid mod menus have.
> 
> This function allows you to load add-on vehicles, weapons, peds, etc.
> 
> Examples: ![image](https://private-user-images.githubusercontent.com/50866332/284281833-9360b3aa-17c9-46af-937b-773ce3879031.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDA5MzUyMjYsIm5iZiI6MTcwMDkzNDkyNiwicGF0aCI6Ii81MDg2NjMzMi8yODQyODE4MzMtOTM2MGIzYWEtMTdjOS00NmFmLTkzN2ItNzczY2UzODc5MDMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI1VDE3NTUyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdlMGJjMjhlYTgzY2Q0YTcyOWJhOTc4M2Y2YmIyZjJhNGRlNWViNDgzYzU3ZGY2MTRiMjE4ZDA0NmY4YTVmNjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.suW6cAEpsoEhVaGDh7BX9tyrn2Mchn6r1BKYXmM33nE)
> 
> ![image](https://private-user-images.githubusercontent.com/50866332/284282003-315599cd-a7df-4325-9831-1b0f676f444c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDA5MzUyMjYsIm5iZiI6MTcwMDkzNDkyNiwicGF0aCI6Ii81MDg2NjMzMi8yODQyODIwMDMtMzE1NTk5Y2QtYTdkZi00MzI1LTk4MzEtMWIwZjY3NmY0NDRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI1VDE3NTUyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVlN2JhN2ZkYzE0MGNjZGI2NWU1MWY5NmNkOGRmZTQ5ZjEwMGZlMjI5OWViMTVmNjJhZDMzZTQ4NTUwYmVkMmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HiL_ouC_427yTM72Hl-16ifcsRwFSavHcpTjlUpElSc)
> 
> And another very interesting function is that it modifies the Hash to be able to join an online session with players while using addons.
> 
> ![image](https://private-user-images.githubusercontent.com/50866332/284282886-0b686fe2-499b-411c-a9e9-fe8ca789cab4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDA5MzUyMjYsIm5iZiI6MTcwMDkzNDkyNiwicGF0aCI6Ii81MDg2NjMzMi8yODQyODI4ODYtMGI2ODZmZTItNDk5Yi00MTFjLWE5ZTktZmU4Y2E3ODljYWI0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI1VDE3NTUyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlZDBmNGIzZGI5ZWFkNWE3OTljZTVlN2NjMjNmYTgwNTEyNGQ2MGEwNTJjZjM5MGUxNzM1NjkzZmNhZDQ2MjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uPrf6cLT6-fMhVpJKZqkqclK4_hT5SZDdDZLEMPGgFY)
> 
> Hopefully someone can implement it ♥

exactly , and we hope someone add that soon
